src/redux/actions.js

```diff
@@ -1,64 +1,91 @@
-import { types } from './types';
-import axios from "axios";
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
 
-export const setContacts = array => ({
-  type: types.SET_CONTACT,
-  payload: array,
-});
+export const createContact = createAsyncThunk(
+  'contacts/createContact',
```

Comment:
Should be moved to the constants here and below

Reply:
Corrected, action titles added to const actionTitles at the beginning of the file.